Example/StripeConnectExample/StripeConnectExample/Views/OptionListRow.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  OptionListRow.swift
+//  StripeConnect Example
+//
+//  Created by Chris Mays on 8/28/24.
+//
+
+import SwiftUI
+
+struct OptionListRow: View {
+    let title: String
+    private(set) var subtitle: String?
+    var selected: Bool = false
+    var onSelected: ()-> Void
+    var body: some View {
+        Button {
+            onSelected()
+        } label: {
+            HStack {
+                VStack(alignment: .leading) {
+                    Text(title)
+                        .font(.body)
+                        .foregroundColor(Color(UIColor.label))
+                    subtitle.map(Text.init)
+                        .font(.caption)
+                        .foregroundColor(Color(UIColor.secondaryLabel))
```

Comment:
I think this is equivalent and more SwiftUI-y
```suggestion
                        .foregroundColor(.primary)
                    subtitle.map(Text.init)
                        .font(.caption)
                        .foregroundColor(.secondary)
```